xdsl/printer.py

```diff
@@ -211,35 +212,32 @@ def _print_operand(self, operand: SSAValue) -> None:
             self.print(" : ")
             self.print_attribute(operand.typ)
 
-    def _print_ops(self, ops: List[Operation]) -> None:
-        self._indent += 1
-        for op in ops:
-            self._print_new_line()
-            self.print_op(op)
-        self._indent -= 1
-        if len(ops) > 0:
-            self._print_new_line()
-
     def print_block_name(self, block: Block) -> None:
         self.print("^")
         if block not in self._block_names:
             self._block_names[block] = self._get_new_valid_block_id()
         self.print(self._block_names[block])
 
-    def print_block(self, block: Block) -> None:
+    def print_block(self, block: Block, print_block_name: bool) -> None:
```

Comment:
We should add a default value. My guess is that the default should be `True`?

Reply:
I'm not sure we want a default value, tbh. It should be fairly rare for people to print blocks as opposed to regions, and the default value seems like a footgun.

Reply:
Added default value